packages/lime-report/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# This is free software, licensed under the GNU General Public License v3.
+#
+
+include $(TOPDIR)/rules.mk
+
+GIT_COMMIT_DATE:=$(shell git log -n 1 --pretty=%ad --date=short . )
+GIT_COMMIT_TSTAMP:=$(shell git log -n 1 --pretty=%at . )
+
+PKG_NAME:=lime-report
+PKG_VERSION=$(GIT_COMMIT_DATE)-$(GIT_COMMIT_TSTAMP)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+	TITLE:=LibreMesh status report utility
+	CATEGORY:=LiMe
+	MAINTAINER:=Ilario Gelmetti <iochesonome@gmail.com>
+	URL:=https://libremesh.org
+	PKGARCH:=all
+endef
+
+define Package/$(PKG_NAME)/description
+	Outputs a report about the status and the main configuration
+	of a LibreMesh node, useful for debugging and issues reporting.
+endef
+
+define Package/$(PKG_NAME)/config
+endef
+
+define Build/Compile
+endef
+
+define Package/$(PKG_NAME)/install
```

Comment:
I don't understand, it already looks good to me, which line should be changed?

Reply:
what marcos says is that `packages/lime-report/files/lime-report.sh` should be something like ` packages/lime-report/files/usr/bin/lime-report.sh`. But then it is not renamed as lime-report so I understand that each solution has pros and cons. In this package that only a bash script exists I would use a `src` directory instead of the `files` directory that in my opinion should be used for full path names (without renaming).